README.md

```diff
@@ -1,2 +1,25 @@
-# demo
-Simple demo for using Utopia + Swoole
+# Demo of utopia-php/swoole
+
+This project uses the [Utopia framework](https://github.com/utopia-php/framework) and [Swoole](https://github.com/utopia-php/swoole) to build a very simple API, demonstrating how to define routes and start the server, both found in the server entrypoint `./app/server.php`.
+
+## Install
+```bash
+git clone https://github.com/kodumbeats/utopiademo
+cd utopiademo
+composer install
```

Comment:
We can have the composer install as part of the first stage of build, same as we do in the Appwrite dockerfile.